src/components/Asset/AssetContent/index.tsx

```diff
@@ -55,8 +55,8 @@ export default function AssetContent({
             )}
 
             <MetaFull ddo={ddo} />
-            {price?.datatoken && <EditHistory />}
-            {price?.datatoken && debug === true && (
+            {price?.datatoken !== null && <EditHistory />}
```

Comment:
this is interesting find, the fine differences between `undefined`, `null`, `0`, `false` in js are always fun to deal with. Just a note that if we have to do more of those checks we should rather implement better returns in our method which creates that `price` object

Reply:
yess, took me some time to understand from where the 00 came from:

![Screenshot from 2022-01-19 14-42-09](https://user-images.githubusercontent.com/49017601/150133198-664cddce-e63f-4884-adb2-3c24c50cf32b.png)

Reply:
right, so I guess we sometime in our refactoring switched to have our `price` object never return undefined but always `0` as a string. I have lots of those `price.datatoken` checks over in #1009 so I'll double check them.

In any case, now that I look at this line I do not know what the purpose of this check is to render `EditHistory` so would try removing that check altogether and just output EditHistory. If something crashes then most likely EditHistory needs to be adapted.